Cinecon/Models.cs

```diff
@@ -25,7 +25,7 @@ public class MenuItem
     {
         public string Name { get; set; }
         // The key is the size/product type. The value is the price.
-        public Dictionary<string, double> ItemTypes { get; set; }
+        public Dictionary<string, decimal> ItemTypes { get; set; }
```

Comment:
Waarom heb je hier decimal van gemaakt?

Reply:
Decimals zijn preciezer dan doubles, waardoor ze beter zijn voor valuta wat we in ons geval ook doen.